config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -13,39 +13,39 @@
 # Rails.application.config.active_storage.track_variants = true
 
 # Apply random variation to the delay when retrying failed jobs.
-Rails.application.config.active_job.retry_jitter = 0.15
+# Rails.application.config.active_job.retry_jitter = 0.15
```

Comment:
> Leave the file for the current version as comments, as that seems to be the convention in all other apps.

Does it? My understanding is this file should be deleted once the `load_defaults` has been set to the new version.

Reply:
The rails docs [recommend](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#configure-framework-defaults) removing the file once it's served it's purpose. I [deleted](https://github.com/alphagov/link-checker-api/pull/479/commits/53a791d5b5406570fc8cf2329dcf3de6d39f3e7c) it on the link-checker-api bump. But I [noticed](https://github.com/search?q=org%3Aalphagov+has_many_inversing+%3D+true&type=code) that it's still in quite a few apps so left it here... Let's get rid! ✂️

Reply:
And three more:

https://github.com/alphagov/finder-frontend/pull/2613
https://github.com/alphagov/search-admin/pull/657
https://github.com/alphagov/support-api/pull/579